litestar/plugins/base.py

```diff
@@ -190,10 +190,27 @@ class OpenAPISchemaPlugin(OpenAPISchemaPluginProtocol):
 
     @staticmethod
     def is_plugin_supported_type(value: Any) -> bool:
-        return False
+        """Given a value of indeterminate type, determine if this value is supported by the plugin.
+
+        This is called by the default implementation of :meth:`is_plugin_supported_field` for
+        backwards compatibility. User's should prefer to override that method instead.
+
+        Args:
+            value: An arbitrary value.
+
+        Returns:
+            A typeguard dictating whether the value is supported by the plugin.
+        """
+        raise NotImplementedError(
+            "One of either is_plugin_supported_type or is_plugin_supported_field should be defined. "
+            "The default implementation of is_plugin_supported_field calls is_plugin_supported_type "
+            "for backwards compatibility. Users should prefer to override is_plugin_supported_field "
+            "as it receives a 'FieldDefinition' instance which is more useful than a raw type."
```

Comment:
```suggestion
            "One of either 'is_plugin_supported_type' or 'is_plugin_supported_field' should be defined. "
            "The default implementation of 'is_plugin_supported_field' calls 'is_plugin_supported_type' "
            "for backwards compatibility. Users should prefer to override 'is_plugin_supported_field' "
            "as it receives a 'FieldDefinition' instance which is more useful than a raw type."
```
to match `FieldDefinition`?

Reply:
how is this outdated already?

Anyway, thanks - I remember thinking this when I wrote it but in the moment the satisfaction of having 3 lines with vertically aligned closing `"` was too strong :laughing:

Reply:
No it was outdated straight after you reviewed.. it wouldn't let me commit the change. I only just make that other commit a few minutes ago.

No worries though I'll fix this before it's merged.